tests/lib/rules/indent.js

```diff
@@ -682,6 +682,12 @@ ruleTester.run("indent", rule, {
                 "  }\n" +
                 "};",
             options: [2]
+        }, {
+            code:
+                "var path     = require('path')\n" +
+                " , crypto    = require('crypto')\n" +
+                " ;\n",
```

Comment:
Why is this unit test passing?
`crypto` has 3 spaces before it? I expect it to have 4.